src/ContentTemplates/Parameters/AssetParameters.php

```diff
@@ -61,7 +61,7 @@ public static function getObjectLabel(): string
     protected function getTargetClasses(): array
     {
         global $CFG_GLPI;
-        return $CFG_GLPI["asset_types"];
+        return $CFG_GLPI["ticket_types"];
```

Comment:
I do not understand why you changed this.

Reply:
Because i have the first php error if the object is not into this array (like PluginBadgesBadge)